extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -31,6 +31,21 @@ export class WorkspaceService implements IWorkspaceService {
 		this.getProjectsInWorkspace(undefined, true).catch(err => Logger.error(`Error initializing projects in workspace ${err}`));
 	}
 
+	get isProjectProviderAvailable(): boolean {
```

Comment:
this was removed in the last PR (it used to be used for determining whether or not the projects view should show), but bringing it back here to use for determining if the new/open project dialogs should be shown.